.github/workflows/github-release.yml

```diff
@@ -47,6 +47,14 @@ jobs:
           path: Mattermost-unsigned.ipa
           name: Mattermost-unsigned.ipa
 
+      -name: ci/sbom-ios
+        uses: ./.github/actions/snyk-sbom
+        working-directory: ./ios
```

Comment:
(echoing my comment in the ticket)

Will this be able to pick up JS dependencies from the root (package*.json)? Same question with android.

Reply:
It should.. npm will check in the parent folder if needed

Reply:
After checking with the generated SBOMs from @enzowritescode, the build-specific SBOMs did not contain `npm` packages. Here is the summary:

```
## sbom_ios.json
- purl (total) -- 178
- pkg:cocoapods -- 170
- pkg:swift -- 8

## sbom_android.json
- purl (total) -- 5953
- pkg:maven -- 5953
```

As compared to SBOM for the whole project:
```
## sbom.json
- purl (total) -- 7546
- pkg:gem -- 138
- pkg:npm -- 1120
- pkg:maven -- 6110
- pkg:cocoapods -- 170
- pkg:swift -- 8
```